### PR TITLE
Adds information about Ana for Keptn's Governance Board Nomination

### PR DESCRIPTION
**Pull request by AnaMMedina21**

### CODE REVIEW

**Comment by agardnerIT** [comment]
Voting is open. Please merge this ASAP.

**Review by thschue — APPROVED** [review]
lgtm

File: elections/2022/governance-committee-candidates.md [hunk 1]
```diff
@@ -48,6 +49,21 @@ When not working as Kubernetes Engineering Director at Canonical; he contributes
 
 Alois works as Chief Technology Strategist at Dynatrace, where he leads all open source and research efforts. He is also co-chair of TAG App Delivery in the CNCF and contributor and maintainer to several other open source projects like OpenFeature and OpenTelemetry. Alois is the initial creator of the Keptn project and was driving most of the conceptual over the last couple of years. He is also a co-creator of the application lifecycle toolkit for Keptn. 
 
+---
+
+### Ana Margarita Medina
+<img src="https://avatars.githubusercontent.com/u/3894791" alt="Ana Margarita Medina" height="200" width="200">
```
Comment (thisthat) [inline elections/2022/governance-committee-candidates.md:55]:
```suggestion

<img src="https://avatars.githubusercontent.com/u/3894791" alt="Ana Margarita Medina" height="200" width="200">

```

**Review by thisthat — APPROVED** [review]

**Comment by thschue** [comment]
> Voting is open. Please merge this ASAP.
The voting is not open at the moment, we're in the nomination period until mid-next week ... Nevertheless, we'll get this merged as soon as possible

**Pull request merged**